src/Downloader.cc

```diff
@@ -58,13 +58,20 @@ DownloaderContext::finished()
     http = nullptr;
 }
 
-void
-Downloader::CbDialer::print(std::ostream &os) const
+std::ostream &
+operator <<(std::ostream &os, const DownloaderAnswer &answer)
 {
-    os << " Http Status:" << status << Raw("body data", object.rawContent(), 64).hex();
+    os << "outcome=" << answer.outcome;
+    if (answer.outcome == Http::scOkay)
+        os << ", resource.size" << answer.resource.length();
+    return os;
 }
```

Comment:
This change also fixes Downloader answer reporting. I am writing this comment to record a possibly generally useful rule of thumb proposal:

Rule of thumb: Functions that print asynchronous call parameters should not dump large objects. Call parameters may be reported several times during the call lifetime, and the corresponding information is likely to be reported by the call sender and/or recipient anyway. Large objects should be reported by the code that creates them and/or by the code that consumes them.